### ISSUE TITLE
Every other search yields 0 results

### ISSUE THREAD

**Issue by rahlzel** [caos30/php_srrFileManager#4]
Love this file manager, but it has a fairly significant flaw.

To duplicate:

1. Enter a search term
2. Click "Search" repeatedly

The results will alternate between X (actual) results and 0 (none) results.

**Comment by caos30** [comment]
Hi rahizel, thanks to take your time for share the bug. But i cannot reproduce it!?
Can you be more explicit/detailed?

I did this:

1. clicked on "Search" at main menu
2. put the word "open" (as example) in the box labelled "String to search:"
3. left marked the default option below "at the files and directories names"
4. clicked on "search now" button
5. then the page is reloaded with n results (es expected)
6. i clicked again "search now" button
7. and the page reload perfectly again the same n results ¿?

So, it seems fine.
I understand that these aren't the steps you toke.
Please, can you explain the differences oni your steps?

Thanks in advance.
Indeed, i don't use this file manager since time long ago... because it really the typical cPanel of the usual servers has good enough (and better) file managers included. But, i'm sure that a light filemanager like this can be useful sometimes. Once i used it for give restricted access to a particular folder to a customer of mine ;) ... for example.

Cheers,
Sergi

**Comment by rahlzel** [comment]
Damn. I think I know the problem, then. It's my fault.

[I commented out this](https://github.com/caos30/php_srrFileManager/blob/master/index.php#L490) with the intention of removing an option I didn't want my group to worry about, knowing that they'd ONLY be using the file content. 

That probably broke something.

**Comment by caos30** [comment]
Ah, ok. So my suggestion for you is that you hide this DIV which contain that searching options! 
You can do modifying the HTML code: replace <div> by <div style='display:none;'> 
I tested it and run fine not breaking nothing ;)

**Comment by rahlzel** [comment]
Sorry, I don't follow. I think you may have forgotten to type something :)

But here's the tricky part (for me): I only want to hide the top radio button option, NOT the bottom one.

**Comment by caos30** [comment]
Yes, you're right... i had a mistake typing here HTML code... it became HIDDEN!! :(
Let me try again. And, indeed, i understood you now.

**Comment by caos30** [comment]
Ok, if you want only show the last radio button, and obviously be marked by default, then:

1. delete the first radio button `<p>` or add to it `style='display:none;'`
2. replace the second radio button for this other:

`<input type='radio' name='where' id='where_2' value='filecontent' checked='true' />`

**Comment by rahlzel** [comment]
Perfect! That works! Thank you!

On a side note, this project is the only light, fast, SEARCH-able PHP file manager I've found. I spent a long time searching through projects here on Github, and in Google, and through Reddit, etc, and the only ones I found had either WAY too many features, or it didn't have file-content search. 

Thank you for this.

**Comment by caos30** [comment]
Yeah... just for this reason i had to make this piece of code! 4 years ago... there was nothing like this. Well, in fact, as i say in the first lines of the index.php file, i built it not from scratch, but taking profit of this other existing project: "osfm Static" project (Devin Smith, July 1st 2003)

http://www.osfilemanager.com

But as you pointed, it hasn't -for example- searching capabilities ;)
So, well... this is the high power of the opensource, it isn't? Make better software from good software :)

Cheers, and thanks for share that info with me! Always is a satisfaction to receive some kind of positive feedback.
